packages/style-stencila/src/variables.css

```diff
@@ -0,0 +1,79 @@
+:root {
```

Comment:
@nokome These are the initial set of variables that I'm proposing and can be configured from Thema as needed.
The font-sizes are commented out, but I think it should give a good range, might want to add `lg/xl` for hero/headlines?

Reply:
Re naming I suggest that we be consistent (internally and with CSS) and use kebab case for everything. e.g. `--font-family-display` is more consistent with CSS `font-family` and with `--font-size-h1`